src/yamlfix/model.py

```diff
@@ -1,17 +1,24 @@
 """Define program entities like configuration value entities."""
 
 from typing import Optional
+from enum import Enum
 
 from maison.schema import ConfigSchema
 
 
+class StyleSequence(Enum):
```

Comment:
Because this can be used for Sequences and Mappings in the future, I'd propose something like `YamlNodeStyle`:

```suggestion
class YamlNodeStyle(Enum):
    """Represent the desired YAML node style for sequences and mappings."""
```